data/human/war reactions.txt

```diff
@@ -168,3 +167,70 @@ mission "War Reactions: Navy and Free Worlds Captains"
 			`	Rain says, "We're uniformed career officers who must follow orders. Our country owns the ships, uniforms, and everything else we use. Our salaries may be meager compared to your earnings, but serving one's country is a rich reward.`
 			`	"You, on the other hand, own your ship and serve whoever you want. To keep up with costs, you have to charge a lot of money for your services. Mercenaries only survive as long as you if they're skilled at both finances and combat."`
 				goto questions
+
+
+mission "War Reactions: Syndicate Orphan"
+	landing
+	minor
+	source
+		government "Syndicate"
+		not attributes "station"
+	to offer
+		has "event: post-war reactions begin"
+		random < 30
+	on offer
+		set "pug media coverage"
+		conversation
+			`	When you exit your ship, you hear sounds of fists falling on skin and a kid yelling for help. Around the corner are three boys hitting a fourth, who just fell on the ground. It's not clear if they're going to stop attacking him.`
+			choice
+				`	"Break it up, kids."`
+				`	(Ignore them and keep walking.)`
+					decline
+			`	The three attackers turn around, looking furious. They eye your gun and run away. "Thanks. I'm Gus," says the fourth boy, as he slowly rises to his feet. He looks about twelve years old.`
+			`	Gus thinks for a second and looks worried. "Can you walk me to the factory? I'm already late for work, and I'm afraid they'll attack me again."`
+			choice
+				`	"Aren't you a little young to work in a factory?"`
+				`	"I'm in a hurry. You're on your own."`
```

Comment:
Might be interesting for Gus to explain why he's attacked? Just before he says he wants you to walk him back to the factory, or have the player ask it (doesn't flow as well).

Reply:
Thank you for noticing that. The player is now informed about the reason, if they ask.